target/linux/ath79/dts/qca9533_dlink_dap-13xx.dtsi

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_green;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_green;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			label = "wps";
+			linux,code = <KEY_WPS_BUTTON>;
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+		};
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_green: power_green {
+			label = "d-link:green:power";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		};
+
+		power_red {
+			label = "d-link:red:power";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi_green {
+			label = "d-link:green:wifi";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi_red {
+			label = "d-link:red:wifi";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+		};
+
+		signal1 {
+			label = "d-link:green:signal1";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		signal2 {
+			label = "d-link:green:signal2";
```

Comment:
d-link:green:rssimax

If you don't like that, you could also use the following scheme:
rssilow
rssimediumlow
rssimediumhigh
rssihigh

The latter is used more frequently

Reply:
This looks like a good idea. Do we need an additional LED migration script in base-files then?
However, though flashing from ar71xx worked (due to `SUPPORTED_DEVICES` entry) I did not manage to connect to the device when choosing to keep settings, not sure if this is a generic problem or could be fixed within the new image.

Reply:
LED migration would be set up here: https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/generic/base-files/etc/uci-defaults/04_led_migration

network setup is always an issue when updating from ar71xx to ath79, that's why we recommend to start with clean config. I haven't actually looked at the config for ar71xx yet, so I can't tell any details. However, there is no entry in 02_network there, so the device actually sets up an eth1 while it doesn't have one:

		ucidef_set_interfaces_lan_wan "eth0" "eth1"